Userland/mount.cpp

```diff
@@ -96,11 +96,18 @@ bool mount_all()
             continue;
         }
 
-        dbg() << "Mounting " << devname << "(" << fstype << ")"
+        const char* filename = parts[0].characters();
+        int fd = open(filename, O_RDONLY);
+        if (fd < 0) {
+            fprintf(stderr, "Failed to open : %s\n", filename);
```

Comment:
Don't forget to print the error itself. Use `perror` or something (except string formatting can change errno, so stach it like this:
```suggestion
            int saved_errno = errno;
            auto message = String::format("Failed to open %s", filename);
            errno = saved_errno;
            perror(message.characters());
```